### PR TITLE
Tyle8552/local data

### PR DESCRIPTION
**Pull request by ghost**
Removes local resources. Adds gradle task to download these to a central location next to the samples project:

/samples-data
/arcgis-runtime-samples-java

### CODE REVIEW

**Comment by ghost** [comment]
@doneill Can you please review these changes?

**Comment by ghost** [comment]
@john0005 just want to make sure you're in the loop on this change

File: build.gradle [hunk 1]
```diff
@@ -10,6 +14,19 @@ allprojects {
     }
 
     dependencies {
-        compile 'com.esri.arcgisruntime:arcgis-java:100.0.0-1450'
+        compile 'com.esri.arcgisruntime:arcgis-java:100.0.0-1459'
     }
 }
+
+// Download data from AGOL for offline data samples
+task downloadData {
```
Comment (doneill) [inline build.gradle:22]: When does this task get run?

File: README.md [hunk 2]
```diff
@@ -24,6 +24,12 @@ On Windows
 
 There is no need to install Gradle to run the samples.
 
+## Offline sample data
+Some samples require offline data. The first time you refresh the Gradle project in an IDE, or run the `build` or 
+`run` tasks, the `downloadData` task will automatically download a local data folder called samples-data into the 
+directory with arcgis-java-runtime-samples. To get the latest data, delete the old `samples-data` folder, and then run
+`downloadData`.
```
Comment (doneill) [inline README.md:31]: I ran the `run` task and the data did not get downloaded.  Prior to this there is no mention of refreshing your Gradle project or running the `build` task.

**Review by doneill — CHANGES_REQUESTED** [review]

**Comment by doneill** [comment]
@tyle8552 Doesn't look like the `downloadData` task ever gets run?  Building or running a sample that depends on local data does not display the data:  

Distance Composite Symbol sample 
`$ ./gradlew :scene:distance-composite-symbol:run --info`

composite-symbol

Find Address sample
`$ ./gradlew :search:find-address:run --info`

find-address

Error: 

```
Exception in thread "JavaFX Application Thread" java.lang.IllegalArgumentException: Parameter boundingGeometry must not be nullnd-address:run
        at com.esri.arcgisruntime.internal.i.e.a(SourceFile:37)
        at com.esri.arcgisruntime.internal.mapping.view.p.a(SourceFile:337)
        at com.esri.arcgisruntime.mapping.view.MapView.setViewpointGeometryAsync(SourceFile:407)
        at com.esri.samples.search.FindAddressSample.lambda$null$1(FindAddressSample.java:142)
        at com.sun.javafx.application.PlatformImpl.lambda$null$173(PlatformImpl.java:295)
        at java.security.AccessController.doPrivileged(Native Method)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$174(PlatformImpl.java:294)
        at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
java.util.concurrent.ExecutionException: com.esri.arcgisruntime.ArcGISRuntimeException: File not found
        at com.esri.arcgisruntime.internal.a.b.get(SourceFile:128)
        at com.esri.samples.search.FindAddressSample$ResultsLoadedListener.run(FindAddressSample.java:291)
        at com.sun.javafx.application.PlatformImpl.lambda$null$173(PlatformImpl.java:295)
        at java.security.AccessController.doPrivileged(Native Method)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$174(PlatformImpl.java:294)
        at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
Caused by: com.esri.arcgisruntime.ArcGISRuntimeException: File not found
        at com.esri.arcgisruntime.ArcGISRuntimeException.createFromInternal(SourceFile:70)
        at com.esri.arcgisruntime.internal.a.b.get(SourceFile:120)
        ... 5 more
```

**Comment by ghost** [comment]
This PR will probably be closed for this one: https://github.com/Esri/arcgis-runtime-samples-java/pull/73

**Pull request merged**